lib/components/molecules/Form/FieldToggle/tests/FieldToggle.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { FieldToggleVanilla } from '../index';
+import { userFormToggleProps } from '../../Form.mock';
+import Form from '../../Form';
+import { shallowWithTheme, mountWithTheme } from '../../../../../../__mock__/themeMock';
+
+describe('<FieldToggle />', () => {
+  let FieldToggleComponent = '';
+
+  beforeEach(() => {
```

Comment:
Why are you using beforeEach as I can see you are creating different wrappers in each test.